ChromaticHQ/calliope
# Upgrade dependencies and configure Dependabot

## Description

This PR upgrades (almost) all dependencies* to their latest major version and adds configuration for Dependabot. I say _almost_ all dependencies because two of them (`chalk` and `del`) are actually upgraded to the next-to-latest major version, both for the same reason: their latest major versions switch to ECMAScript Modules-only support. In order to upgrade to those newest versions of `chalk` and `del`, we would need to invest more effort than I currently have capacity for.

The PR also replaces [`node-sass`](https://npmjs.com/package/node-sass) with [`dart-sass`](https://npmjs.com/package/dart-sass), the latest (and recommended) version of Sass, as suggested by the message at the top of `node-sass`’s README.

## Motivation / Context

While attempting to reproduce an issue I noticed in CHQ, I realized that our dependencies were woefully out of date. It appears I never configured Dependabot on this project. 😬

## Testing Instructions / How This Has Been Tested

All checks should pass. I will cut a new major release of this library and wait for Dependabot to submit a PR for `chromatichq-eleventy` and see if we get any regressions.

## Screenshots

n/a, but here’s a re-enactment of me trying to act normal when I realized what I had done (or failed to do, as it were):

![act-normal](https://user-images.githubusercontent.com/439649/216431819-cf4bfa1b-3d65-4e7f-8b4d-43ed41c20502.gif)

## Documentation

n/a
